boards/DIM/Test/Src/Test_Dim.cpp

```diff
@@ -1,3 +1,35 @@
 #include "Test_Dim.h"
 
 DEFINE_FFF_GLOBALS;
+
+void DimTest::SetUp()
```

Comment:
Take advantage of defining a common Setup() and Teardown() in the base class to reduce code bloat.